### PR TITLE
tiny OOP bug fix

### PR DESCRIPTION
**Pull request by dcolinmorgan**
OOP/self bug, fixed so panda_network==force

### CODE REVIEW

File: netZooPy/panda/panda.py [hunk 1]
```diff
@@ -462,7 +456,7 @@ def gupdate_diagonal(diagonal_matrix, num, alpha, step):
                 correlation_matrix=cp.array(correlation_matrix)
                 W = 0.5 * (gt_function(ppi_matrix, motif_matrix) + gt_function(motif_matrix, correlation_matrix))  # W = (R + A) / 2
                 hamming = cp.abs(motif_matrix - W).mean()
-                motif_matrix=cp.array(motif_matrix)
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:465]: why you deleted this @dcolinmorgan

Comment (dcolinmorgan) [reply netZooPy/panda/panda.py:465]: it was redundant with 456

File: netZooPy/panda/panda.py [hunk 2]
```diff
@@ -43,7 +38,7 @@ class Panda(object):
     Outputs:
 
      Authors: 
-       cychen, davidvi, alessandromarin, Marouen Ben Guebila, Daniel Morgan
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:46]: and this one

File: netZooPy/panda/panda.py [hunk 3]
```diff
@@ -30,11 +30,6 @@ class Panda(object):
                     (Default)'union': takes the union of all TFs and genes across priors and fills the missing genes in the priors with zeros.
                     'intersection': intersects the input genes and TFs across priors and removes the missing TFs/genes.
         remove_missing: removes the gens and TFs that are not present in one of the priors. Works only if modeProcess='legacy'
-        computing  : 'cpu' uses Central Processing Unit (CPU) to run PANDA
```
Comment (marouenbg) [inline netZooPy/panda/panda.py:33]: and L33 -> l37 too

File: netZooPy/panda/panda.py [hunk 4]
```diff
@@ -507,7 +512,7 @@ def gupdate_diagonal(diagonal_matrix, num, alpha, step):
             tfs = np.tile(self.unique_tfs, (len(self.gene_names), 1)).flatten()
             genes = np.repeat(self.gene_names,self.num_tfs)
             motif = self.motif_matrix_unnormalized.flatten(order='F')
-            force = motif_matrix.flatten(order='F')
+            force = self.motif_matrix.flatten(order='F')
```
Comment (marouenbg) [inline netZooPy/panda/panda.py]: @dcolinmorgan why did you add `self` here?

Comment (dcolinmorgan) [reply netZooPy/panda/panda.py]: FUCK
